webcompat/static/js/lib/models/issue.js

```diff
@@ -91,13 +91,10 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var el = body;
-    el = $("strong:contains('Description')", el);
-    if (el[0]) {
-      var description = el[0].nextSibling.textContent;
-      description = description.replace(": ", "");
-      return description != null ? description.slice(0, 74) : null;
-    }
+    body = document.createRange().createContextualFragment(body).textContent;
```

Comment:
(this is a little weird, even if we wanted to go down this route, we could just create a `DocumentFragment` directly: https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment